Makefile

```
@@ -782,3 +786,23 @@ ifdef MLRUN_DOCKER_CACHE_FROM_TAG
 	done;
     MLRUN_DOCKER_CACHE_FROM_FLAG := $(MLRUN_BASE_IMAGE_DOCKER_CACHE_FROM_FLAG)
 endif
+
+.PHONY: verify-uv-version
+verify-uv-version:
+	@{ \
+	uv_version=$$(uv version | cut -d' ' -f2); \
+	result=$$(pysemver compare $$uv_version $(MLRUN_UV_VERSION)); \
```

```suggestion
	result=$$(python -m semver compare $$uv_version $(MLRUN_UV_VERSION)); \
```

While this is a suggestion, the reasons for it
1. explicitness
2. indicates exactly which python runs the cli command

---

Done in b5db017